packages/Card/src/components/Title/Title.js

```diff
@@ -5,19 +5,21 @@ import * as sc from "./Title.styles";
 const propTypes = {
   /** Body content of the Title. */
   children: PropTypes.node,
+  /** Semantic level of Heading for the title */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
 };
 
 const defaultProps = {
   children: null,
+  level: 2,
 };
 
 const Title = props => {
-  const { children } = props;
-
+  const { children, level } = props;
   return (
-    <sc.titleStyles data-pka-anchor="card.title" {...props}>
+    <sc.Title data-pka-anchor="card.title" level={level}>
```

Comment:
```suggestion
    <sc.Title data-pka-anchor="card.title" level={2} {...moreProps}>
```

Reply:
Then you don't even need to add a `level` prop to this component 🤯 

Then again, it might be a good idea to leave it, so it's an explicit part of the API... but you could also just find a way to add a note to the `README` / `Docs` page that explains that the `<Card.Title>` includes all the same props as the `<Heading>`.